client/src/components/GameCreator.tsx

```diff
@@ -51,24 +51,29 @@ export function GameCreator(props: GameCreatorProps) {
         selected={winningScore}
         onSelect={(s) => setWinningScore(Number(s))}
       />
-      <button
-        className="mb-3"
-        onClick={async () => {
-          if (!isLoading) {
-            setIsLoading(true);
-            try {
-              const lobby = await getLobby(lobbyClient, playerToken, region, initialConfig, visibility);
-              joinRoom(lobby.roomId);
-            } catch (e) {
-              setIsLoading(false);
-              setError(e instanceof Error ? e.toString() : typeof e === "string" ? e : "Unknown error");
+      <div className={"mb-3 flex items-center justify-center"}>
+        <button
+          onClick={async () => {
+            if (!isLoading) {
+              setIsLoading(true);
+              try {
+                const lobby = await getLobby(lobbyClient, playerToken, region, initialConfig, visibility);
+                joinRoom(lobby.roomId);
+              } catch (e) {
+                setIsLoading(false);
```

Comment:
nit: can we put setIsLoading in a finally block instead of a catch block? i know it doesn't matter because we redirect immediately but i think it would be nice for consumers who might not do that